tests/test_ota_metadata/test_file_table/test__orm.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2022 TIER IV, INC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from __future__ import annotations
+
+import pytest
+
+from ota_metadata.file_table._types import FileEntryAttrs
+
+
+@pytest.mark.parametrize(
+    "_in",
+    (
+        # directory
+        (
+            FileEntryAttrs(
+                mode=0o040755,
+                uid=1000,
+                gid=1000,
+                xattrs={
+                    "user.foo": "foo",
+                    "user.bar": "bar",
+                },
+            )
+        ),
+        # normal file
+        (
+            FileEntryAttrs(
+                mode=0o100644,
+                uid=1000,
+                gid=1000,
+                size=12345,
+                inode=67890,
```

Comment:
By the way, do we have the inode limit in the current system?

Reply:
> By the way, do we have the inode limit in the current system?

Ah, the inode here is for identifying hardlinked files in group, this value itself is meaningless, it is just for identifying hardlinked files group.

Image we have 5 files having the same digest `0xaabbcc`, but 3 of them belongs to one hardlink group(pointing to the same inode with id 111), 2 of them belongs to another hardlink group(pointing to the same inode with id 222). 
Here we can use the inode(111 and 222) to identify these two groups, but not actually using the inode values.